cemu/tools/chocolateyinstall.ps1

```diff
@@ -4,22 +4,27 @@ $packageName = 'cemu'
 $url32       = 'http://cemu.info/releases/cemu_1.15.17.zip'
 $checksum32  = '317a34a15765e411d476464338d210eeca16b36c3e1401ea1f2096b0fa5a159f'
 $toolsPath   = Split-Path $MyInvocation.MyCommand.Definition
-$cemu_folder = "$toolsPath\cemu"
+$cemu_folder = "$(Get-ToolsLocation)\cemu"
 
 $packageArgs = @{
   packageName    = $packageName
-  url          = $url32
+  url            = $url32
   checksum       = $checksum32
   checksumType   = 'sha256'
   unzipLocation  = $toolsPath
 }
 
+If (Get-Item "$env:ChocolateyInstall\lib-bkp\cemu\tools\cemu\Cemu.exe" -ErrorAction SilentlyContinue) {
+	Write-Output "Migrating cemu location to $cemu_folder"
+	Move-Item "$env:ChocolateyInstall\lib-bkp\cemu\tools\cemu" "$(Get-ToolsLocation)" -Force
```

Comment:
chocolatey moves the entire folder to bkp after beforemodify.ps1 runs, but before chocolateyinstall.ps1. beforemodify from the previous version is used, so it is useless to us here.